challenge.rb

```diff
@@ -1,26 +1,26 @@
 
 def capitalize_each_string(input)
-  #implement your solution here
+  return input.map{|x| x.capitalize}
```

Comment:
remember there's no need for explicit returns here.

Reply:
It's more common and preferred to write this:

```ruby
def capitalize_each_string(input)
  input.map(&:capitalize)
end
```

See:
http://www.brianstorti.com/understanding-ruby-idiom-map-with-symbol/
https://stackoverflow.com/questions/14881125/what-does-to-proc-method-mean